hyperstart/api/grpc/hyperstart.proto

```diff
@@ -5,15 +5,17 @@ package grpc;
 import "google/protobuf/empty.proto";
 
 // To generate hyperstart.pb.go, run:
-// $ protoc --go_out=plugins=grpc:. hyperstart.proto
+// $ go get -u github.com/google/protobuf
+// $ protoc -I=$GOPATH/src/github.com/google/protobuf/src/ -I. --go_out=plugins=grpc:. hyperstart.proto
 
 // unstable
 service HyperstartService {
 	// execution
-	rpc AddContainer(AddContainerRequest) returns (google.protobuf.Empty);
-	rpc AddProcess(AddProcessRequest) returns (google.protobuf.Empty);
-	rpc SignalProcess(SignalProcessRequest) returns (google.protobuf.Empty);
-	rpc WaitProcess(WaitProcessRequest) returns (WaitProcessResponse); // wait & reap like waitpid(2)
+	rpc CreateContainer(CreateContainerRequest) returns (google.protobuf.Empty);
+	rpc StartContainer(StartContainerRequest) returns (google.protobuf.Empty);
+	rpc Exec(ExecRequest) returns (google.protobuf.Empty);
+	rpc Signal(SignalRequest) returns (google.protobuf.Empty);
+	rpc Wait(WaitRequest) returns (WaitResponse); // wait & reap like waitpid(2)
```

Comment:
The names with "Process" would be clearer.
How about ExecProces()(renamed from AddProcess()) SignalProcess() WaitProcess()(unchanged)?

Reply:
@laijs Fine with me. Done.